adapters/adapterstest/test_json.go

```diff
@@ -65,7 +66,12 @@ func RunJSONBidderTest(t *testing.T, rootDir string, bidder adapters.Bidder) {
 func runTests(t *testing.T, directory string, bidder adapters.Bidder, allowErrors, isAmpTest, isVideoTest bool) {
 	if specFiles, err := os.ReadDir(directory); err == nil {
```

Comment:
Did you consider using `filepath.Glob` to use a wildcard pattern in the file listing?

Reply:
We were able to remove this function entirely in the latest delta. Let me know if you agree with the approach.